aten/src/ATen/native/Convolution.cpp

```diff
@@ -495,6 +495,13 @@ at::Tensor convolution(
                           ctx.benchmarkCuDNN(), ctx.deterministicCuDNN(), ctx.userEnabledCuDNN());
 }
 
+at::Tensor convolution_generic(
+    const Tensor& input, const Tensor& weight, const Tensor& bias,
+    IntArrayRef stride, IntArrayRef padding, IntArrayRef dilation,
+    bool transposed, IntArrayRef output_padding, int64_t groups) {
+  AT_ERROR("Backend should either override this function or use thnn implementation");
```

Comment:
I'd probably like it if we gave a more complete error message here XD

Reply:
Basically, the only time you'll trigger this error is if you're working on a backend implementation. So we could give guidance (or a link to guidance) about what to do in this situation.